api/candlepin-api-spec.yaml

```diff
@@ -2192,6 +2192,309 @@ paths:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: An owner could not be found using the provided key
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+    put:
+      description: |
+        Updates an owner.
+        Note: To un-set the defaultServiceLevel for an owner, submit an empty string.
+      tags:
+        - Owner
+      operationId: updateOwner
+      security: []
+      parameters:
+        - name: owner_key
+          in: path
+          description: The key of the owner
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Owner to be updated
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/OwnerDTO'
+      responses:
+        200:
+          description: Owner successfully updated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OwnerDTO'
+        400:
+          description: |
+            Invalid inputs, Owner cannot be updated.
+            Reasons could be, invalid key
+            or the default service level is specified
+            or the owner content access mode and content access mode list cannot be set directly in standalone mode
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: |
+                  Invalid inputs, Owner cannot be updated.
+                  Reasons could be, invalid key
+                  or the default service level is specified
+                  or the owner content access mode and content access mode list cannot be set directly in standalone mode
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+    delete:
+      description: Removes an owner
+      tags:
+        - Owner
+      operationId: deleteOwner
+      parameters:
+        - name: owner_key
+          in: path
+          description: The owner key
+          required: true
+          schema:
+            type: string
+        - name: revoke
+          in: query
+          description: Boolean value to revoke an owner
+          schema:
+            type: boolean
+            default: true
+        - name: force
+          in: query
+          description: Boolean value to forcefully delete an owner. This is not in use.
+          schema:
+            type: boolean
+            default: false
+      security: []
+      responses:
+        200:
+          description: Owner successfully deleted
+        409:
+          description: Occured conflict while owner deletion
```

Comment:
You might consider changing the description to ``` Conflict occurred while deleting an owner ```

Reply:
Updated.